scripts/download-resources.mjs

```diff
@@ -80,6 +102,12 @@ export async function download(url, destPath, overwrite = false, access = fs.con
   }
 }
 
+export async function getResource(url) {
+  const response = await got(url);
```

Comment:
We already depend on `node-fetch`; if it's not too much work (it looks like it's pretty close), would it be possible to reuse that instead of using a separate package?

Reply:
done